src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss

```diff
@@ -0,0 +1,83 @@
+@import "mixins";
+@import "variables";
+
+$row-height: 70px;
+
+:host {
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    margin-bottom: $basic-padding;
+    padding: 0 $basic-padding * 2;
+    max-height: $row-height;
+}
+
+.dot-add-rows__container {
+    border: 1px solid $gray-light;
+    overflow: hidden;
+    height: $row-height;
+    position: relative;
+
+    .dot-add-rows-button__container, .dot-add-rows-columns-list__container {
+        display: flex;
+        position: absolute;
+        width: 100%;
+        transition: top 200ms ease-in-out, bottom 200ms ease-out;
+    }
+
+    .dot-add-rows-button__container {
+        text-align: center;
+        height: $row-height;
+        align-items: center;
+        justify-content: center;
+        top: -70px;
```

Comment:
You can do ```top: -#{$row-height}```, check this sass interpolation variables for more info: http://krasimirtsonev.com/blog/article/SASS-interpolation-in-a-name-of-variable-nest-variables-within-variables